tests/test_base.py

```diff
@@ -40,36 +44,51 @@ def lookup(cls, username):
 
 class TestPraetorian:
 
-    def test_encrypt_password(self, app, user_class):
+    def test_hash_password(self, app, user_class):
         """
         This test verifies that Praetorian encrypts passwords using the scheme
         specified by the HASH_SCHEME setting. If no scheme is supplied, the
         test verifies that the default scheme is used. Otherwise, the test
         verifies that the encrypted password matches the supplied scheme.
         """
+        stock_config = app.config.copy()
+
         default_guard = Praetorian(app, user_class)
-        secret = default_guard.encrypt_password('some password')
+        secret = default_guard.hash_password('some password')
         assert default_guard.pwd_ctx.identify(secret) == 'pbkdf2_sha512'
 
+        # modify to allow 'plaintext' which we'd never support for use
+        """
+        app.config['PRAETORIAN_HASH_DEPRECATED_SCHEMES'] = ['md5_crypt']
+        app.config['PRAETORIAN_HASH_ALLOWED_SCHEMES'] = ['plaintext']
         app.config['PRAETORIAN_HASH_SCHEME'] = 'plaintext'
         dumb_guard = Praetorian(app, user_class)
-        assert dumb_guard.encrypt_password('some password') == 'some password'
+        assert dumb_guard.hash_password('some password') == 'some password'
+        """
+
+        # reset to defaults
+        app.config = stock_config.copy()
```

Comment:
This would be a great fixture to add in to conftest. Something that fetches the app's config and resets it after a function. I think I'll add that in when I get a chance if you don't want to

Reply:
submitted